test/integration/override_default/serverspec/default_spec.rb

```diff
@@ -3,22 +3,25 @@
 describe 'non-standard elasticsearch install and configure' do
   path_component = rhel? ? 'sysconfig' : 'default'
 
-  it_behaves_like 'elasticsearch user',     user: 'foo',
-                                            uid: 1111,
-                                            shell: '/bin/sh',
-                                            group: 'bar',
-                                            gid: 2222
+  it_behaves_like 'elasticsearch user', user: 'elasticsearch',
```

Comment:
The original tests here assumed defaults like user: 'elasticsearch' if they weren't provided. I think we should be testing the overrides as well as the original values from the deleted files (e.g. test/integration/chef-12.14.89/serverspec/default_spec.rb). What's the reason for no longer testing the various attribute overrides work?

Reply:
Copying in what, @hrak did on #737 . The intent wasn't clear why they were used so I defaulted them.